misc/config_tools/schema/VMtypes.xsd

```diff
@@ -178,22 +177,32 @@ CLOSID 0 and the second is mapped to virtual CLOSID 1, etc.</xs:documentation>
   </xs:annotation>
   <xs:restriction base="xs:string">
     <xs:enumeration value="legacy" />
+      <xs:annotation acrn:title="Legacy" />
```

Comment:
The `xs:annotation` node shall be a child of `xs:enumeration`, i.e.

```
<xs:enumeration value="legacy">
  <xs:annotation acrn:title="Legacy" />
</xs:enumeration>
```

Reply:
Done. Let's try building again.